historical-bank.gemspec

```diff
@@ -44,7 +44,7 @@ Gem::Specification.new do |s|
 
   s.add_runtime_dependency 'money',    '~> 6.7'
   s.add_runtime_dependency 'httparty', '~> 0.14'
-  s.add_runtime_dependency 'redis',    '~> 3.3'
+  s.add_runtime_dependency 'redis',    '>=3.3', '< 4.1'
```

Comment:
doesn't `'>=3.3', '< 4.1'` need to be enclosed in `[]`? When I do `bundle install`, it's picking up the `3.3` version, not `4.0.1` as it does when you enclose it in brackets. I'm afraid it's just ignoring the last string, it needs to be an array